src/lib/chat/matrix-client.ts

```diff
@@ -36,6 +36,12 @@ enum ConnectionStatus {
   Disconnected = 'disconnected',
 }
 
+enum MembershipStateType {
+  Invite = 'invite',
+  Join = 'join',
+  Leave = 'leave',
+}
+
```

Comment:
might be worth moving these to `utils/types.ts`?

Reply:
i think we can just use `chat/matrix/types.ts`. OR `chat/types.ts` works as well, but if we have more types/enums (matrix related), we may need  to eventually move to `chat/matrix/types.ts`...

Reply:
Nice sounds good! I added a `chat/matrix/types.ts` 👍